src/services/accountService.js

```diff
@@ -21,14 +22,25 @@ export const completeRegistration = async ({ logo, bio, contacts }) => {
         const json = await res.json();
 
         if (!res.ok) {
+
+            createEvent(EVENT_TYPES.ERROR(
+                "Complete Registration",
+                "BAD_RESPONSE",
+                res.status
+            ));
+
```

Comment:
We also want to have "success" metrics (after this if). With this we would know how many registrations we gain over time and check possible trends

Reply:
Again, success metrics apply to other services as well